lib/index.js

```diff
@@ -0,0 +1,122 @@
+    return typeof hoc === 'function';
+  }); // clear & set
+};
+
+var enhance = function enhance(component) {
+  return config.enhancers.reverse().reduce(function (x, f) {
+    return f(x);
+  }, component);
+};
+
+var renderReactWithAphrodite = exports.renderReactWithAphrodite = function renderReactWithAphrodite(name, component) {
+  var enhancedComponent = enhance(component);
+  return (0, _hypernova2['default'])({
+    server: function () {
+      function server() {
+        return function (props) {
+          var _StyleSheetServer$ren = _aphrodite.StyleSheetServer.renderStatic(function () {
+            var element = _react2['default'].createElement(enhancedComponent, props);
+            return _server2['default'].renderToString(element);
+          }),
+              html = _StyleSheetServer$ren.html,
+              css = _StyleSheetServer$ren.css;
+
+          var style = '<style data-aphrodite="data-aphrodite">' + String(css.content) + '</style>';
+          var markup = (0, _hypernova.serialize)(name, html, props);
+          var classNames = (0, _hypernova.toScript)({ 'aphrodite-css': name }, css.renderedClassNames);
+
+          return style + '\n' + String(markup) + '\n' + String(classNames);
+        };
+      }
+
+      return server;
+    }(),
+    client: function () {
+      function client() {
+        var classNames = (0, _hypernova.fromScript)({ 'aphrodite-css': name });
+        if (classNames) _aphrodite.StyleSheet.rehydrate(classNames);
+
+        var payloads = (0, _hypernova.load)(name);
+        if (payloads) {
+          payloads.forEach(function (payload) {
+            var node = payload.node,
+                data = payload.data;
+
+            if (node) {
+              var element = _react2['default'].createElement(enhancedComponent, data);
+              _reactDom2['default'].render(element, node);
+            }
+          });
+        }
+
+        return component;
+      }
+
+      return client;
+    }()
+  });
+};
+
+var renderReactWithAphroditeStatic = exports.renderReactWithAphroditeStatic = function renderReactWithAphroditeStatic(name, component) {
+  var enhancedComponent = enhance(component);
+  return (0, _hypernova2['default'])({
+    server: function () {
+      function server() {
+        return function (props) {
+          var _StyleSheetServer$ren2 = _aphrodite.StyleSheetServer.renderStatic(function () {
+            var element = _react2['default'].createElement(enhancedComponent, props);
+            return _server2['default'].renderToStaticMarkup(element);
+          }),
+              html = _StyleSheetServer$ren2.html,
+              css = _StyleSheetServer$ren2.css;
+
+          var style = '<style data-aphrodite="data-aphrodite">' + String(css.content) + '</style>';
+
+          return style + '\n' + String(html);
+        };
+      }
+
+      return server;
+    }(),
+    client: function () {
+      function client() {}
+
+      return client;
+    }()
+  });
+};
```

Comment:
this is just here temporarily

Reply:
this is build output; I'd expect the entire `lib` folder to be gitignored, Did you `git add -f` this? (is this so you can reference the branch?)

Reply:
Exactly! I added lib/index.js here just for testing. 

I'm thinking it would be best to just merge this PR rather than go about it the roundabout way. What do you think?